yellowbrick/utils.py

```diff
@@ -184,6 +184,28 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+def is_structured_array(obj):
```

Comment:
Nice - potentially in the future we could create a single function, `has_named_columns` that does `is_dataframe() or is_structured_array()` since this seems like it will be the common use-case.